lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1409,7 +1425,9 @@ public function getSqlResultSetMapping($name)
     /**
      * Gets all sql result set mappings of the class.
      *
-     * @psalm-return array<string, array<string, mixed>>
+     * @return (array|string)[][]
```

Comment:
:thinking: phpcs might be happy with this for now, but maybe this should become `(mixed[]|string)[][]` or something more precise to be future proof

Reply:
fixed